subworkflows/nf-core/utils_nfcore_pipeline/tests/main.function.nf.test

```diff
@@ -121,4 +118,23 @@ nextflow_function {
         }
 
     }
+
+    test("Test Function with logColours") {
+        function "logColours"
+
+        when {
+            function {
+                """
+                // input[0] = true
```

Comment:
uncomment and flip to test log colours with `monochrome_logs=false` matching test description
```suggestion
                input[0] = false
```